spearhead/assets/sass/_responsive.scss

```diff
@@ -0,0 +1,137 @@
+/**
+ * Repsonsive Styles
+ */
+
+/**
+ * Required Variables
+ */
+$flexwidth: 100%;
+$horizontal_space: var(--global--spacing-horizontal);
+
+$breakpoint_sm: 482px;
+$breakpoint_md: 592px;
+$breakpoint_lg: calc( 780px + 48px ); // default width + the margins
+$breakpoint_xl: calc( 1247px + 48px ); // wide width + the margins
+$breakpoint_xxl: 1440px;
+
+// Responsive breakpoints mixin
+@mixin media( $res ) {
```

Comment:
I have a few questions about this:
1. Do we need to define this mixin again? Could we reuse the code from the parent theme and just redefine the variables?

2. Could we use the mixin from core?
https://github.com/WordPress/gutenberg/blob/master/packages/base-styles/_mixins.scss

Reply:
1. I think so, we would just need to extract the mixin itself from the [`_responsive_logic.scss`](https://github.com/Automattic/themes/blob/master/seedlet/assets/sass/structure/_responsive-logic.scss#L18-L79).

2. I could see a benefit to doing this if we used the same breakpoints as core also.